### ISSUE TITLE
[INSTALL]: EPIC please install the latest crtm/2.4.0.1 on all RDHPCS machines

### ISSUE THREAD

**Issue by Hang-Lei-NOAA** [JCSDA/spack-stack#901]
### Package name

crtm

### Package version/tag

2.4.0.1

### Build options

Need to update the fix files with certain option

### Installation timeframe

as soon as possible

### WCOSS2

- [ ] Check this box if and only if your package should be installed on WCOSS2 Cactus and Dogwood (all spack-stack packages will be installed on Acorn). If not, you may disregard the rest of the items below and submit this request.

### WCOSS2: Technical & security review list

- [ ] The code is mature, stable, and production ready
- [ ] The code is does not and cannot use the internet, and does not contain URLs (http, https, ftp, etc.) except in comments
- [ ] The package does not contain prebuilt binary files that have not been approved by NCO security review
- [ ] The code has no publicly disclosed cybersecurity vulnerabilities and exposures (search https://cve.mitre.org/cve/)
- [ ] The code is not prohibited by DHS, DOC, NOAA, or NWS
- [ ] The code comes from a trusted source. Trusted sources include other NWS, NOAA, or DOC, agencies, or other Federal agencies that operate at a FISMA high or equivalent level. Additionally, trusted sources could be third-party agencies through which there is an existing SLA on file (such as RedHat).
- [ ] The code is actively maintained and supported (it continues to get updates, patches, etc.)
- [ ] The code is not maintained by a private entity operating in a foreign country (if it is, make a note below)
- [ ] There is sufficient documentation to support maintenance
- [ ] There are no known security vulnerabilities or weaknesses
- [ ] Installing and running the code does not require privileged processes/users
- [ ] There are no software dependencies that are unapproved or have security concerns (if there are, make a note below)
- [ ] There are no concerns related to SA, SI, and SC NIST control families

**Comment by climbfuji** [comment]
Yes, this will be part of spack-stack-1.6.0.

**Comment by DavidHuber-NOAA** [comment]
It looks like the FTP path for the CRTM fix files is not whitelisted on Hera.  The following command stalls and eventually times out:

```
[David.Huber@hfe01 SPACK]$ wget ftp://ftp.ssec.wisc.edu/pub/s4/CRTM/fix_REL-2.4.0_emc_07112023.tgz
--2023-12-13 20:52:46--  ftp://ftp.ssec.wisc.edu/pub/s4/CRTM/fix_REL-2.4.0_emc_07112023.tgz
           => 'fix_REL-2.4.0_emc_07112023.tgz'
Resolving ftp.ssec.wisc.edu (ftp.ssec.wisc.edu)... 144.92.1.6
Connecting to ftp.ssec.wisc.edu (ftp.ssec.wisc.edu)|144.92.1.6|:21...
```

**Comment by Hang-Lei-NOAA** [comment]
wget was forbidden on Hera.
I have clearly stated in another email to Jong and Natalie. for getting the
fix file and transferred to Hera before building it.

**Comment by climbfuji** [comment]
Installation is now complete - `spack-stack@1.6.0` is available everywhere and has `crtm@2.4.0.1`
